### PR TITLE
Sortable Markdown Table Edits

### PR DESCRIPTION
**Pull request by utku-ozturk**
1. Sorting abilities are improved and simplified: ([screencast](https://trello-attachments.s3.amazonaws.com/5d8f8b1b9c43fb39fe46a9c6/5f6373822790091b2a4bcb1b/dcf64c2c576395264f02b8cf068d3cf3/659497f958891b63171f5f3d020c3047.gif))

- Natural sorting support is added to non-numeric columns.

e.g. In the previous version, if you sort "7 in situ Hi-C" and "58 in situ Hi-C" in ascending order, you get:

```
58 in situ Hi-C
7 in situ Hi-C
```

If natural sorting (w/ numeric option is true) is applied then you'll get

```
7 in situ Hi-C
58 in situ Hi-C
```

- MD table column sorting is aligned to existing browse/search tables column sorting: only one column is allowed to be sorted when the user clicks in column header sorting is done in ascending-descending-initial order.

2. Horizontal scrolling of header and body is synchronized. ([screencast](https://trello-attachments.s3.amazonaws.com/5d8f8b1b9c43fb39fe46a9c6/5f6373822790091b2a4bcb1b/a5c69a66dcede1ed06722c6e2207c2b9/62b1f6e7b238a87bd4086221bcd2fdc1.gif))

3. New (optional) `maxContainerHeight` prop is introduced to set max height. ([screencast](https://trello-attachments.s3.amazonaws.com/5d8f8b1b9c43fb39fe46a9c6/5f6373822790091b2a4bcb1b/913a57e729c7be59818491929ad5268c/39bb17754a7efd423d9397ed392f766f.gif))

### CODE REVIEW

File: src/encoded/static/components/static-pages/placeholders/MdSortableTable.js [hunk 1]
```diff
@@ -596,14 +605,15 @@ function SortableTableRow(props) {
 }
 
 function SortableTableBody(props) {
-    const { data, columns, widths, fullRowWidth } = props;
+    const { data, columns, widths, fullRowWidth, maxContainerHeight } = props;
```
Comment (j1z0) [inline src/encoded/static/components/static-pages/placeholders/MdSortableTable.js:608]: Codacy found an issue: ['maxContainerHeight' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201&bid=20294178)

**Review by alexkb0009 — APPROVED** [review]
Looks good!

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201&bid=20294178)

File: src/encoded/static/components/static-pages/placeholders/MdSortableTable.js (hunk 1 above)
Comment (j1z0) [inline src/encoded/static/components/static-pages/placeholders/MdSortableTable.js:608]: Codacy found an issue: ['maxContainerHeight' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201&bid=20294178)

File: src/encoded/static/components/static-pages/placeholders/MdSortableTable.js [hunk 2]
```diff
@@ -596,14 +628,15 @@ function SortableTableRow(props) {
...
```
Comment (j1z0) [inline src/encoded/static/components/static-pages/placeholders/MdSortableTable.js:631]: Codacy found an issue: ['maxContainerHeight' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201)

File: src/encoded/static/components/static-pages/placeholders/MdSortableTable.js [hunk 3]
```diff
@@ -53,7 +54,7 @@ export class MdSortableTable extends React.PureComponent {
     }
 
     render() {
-        const { defaultColWidths } = this.props;
+        const { subTitle, defaultColWidths, maxContainerHeight } = this.props;
```
Comment (j1z0) [inline src/encoded/static/components/static-pages/placeholders/MdSortableTable.js:57]: Codacy found an issue: ['subTitle' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6227201)

**Pull request merged**